code/modules/cargo/packs.dm

```diff
@@ -2170,6 +2170,48 @@
 					)
 	crate_name = "grilling fuel kit crate"
 
+/datum/supply_pack/organic/beefbroth
+
+	name = "Beef Broth Bulk Crate"
+	desc = "Noone really wants to order beef broth so were selling it in bulk!"
+	cost = 15000
+	contraband = TRUE
+	crate_type = /obj/structure/closet/crate
+	contains = list(/obj/item/reagent_containers/food/snacks/canned/beefbroth,
```

Comment:
Tone this down quite a bit, there shouldn't be more than 4-8 at the most.

Reply:
yes but 32 cans of beef broth is funny